src/TransitionGroup.js

```diff
@@ -73,10 +76,6 @@ class TransitionGroup extends React.Component {
       }
     }
 
-    // If we want to someday check for reordering, we could do it here.
```

Comment:
This looks like a TODO comment, we should keep it unless there is a reason to remove it.